contributed_definitions/nyaml/NXcorrector_cs.yaml

```diff
@@ -1,42 +1,45 @@
 category: base
 doc: |
-  Corrector for aberrations in an electron microscope.
+  Base class for a corrector reducing (spherical) aberrations in electron microscopy.
   
   Different technology partners use different naming schemes and
   models for quantifying the aberration coefficients.
   
   The corrector in an electron microscope is composed of multiple lenses
   and multipole stigmators with details specific for the technology partner
   and microscope. Many of their technical details is proprietary knowledge.
+  
+  If functionalities for correcting multiple aberrations are included in
+  one :ref:`NXcomponent_em` `like it is reported here <https://www.ceos-gmbh.de/en/research/electrostat>`_ 
+  use multiple groups:
+  
+  * :ref:`NXcorrector_cs` for spherical aberration
+  * :ref:`NXmonochromator_em` for energy filtering or chromatic aberration
+  * :ref:`NXcorrector_ax` for axial astigmatism correction
+  
 type: group
 symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
   n_img: |
     Number of images taken, at least one.
 NXcorrector_cs(NXcomponent_em):
+  # user perspective
   applied(NX_BOOLEAN):
     doc: |
       Was the corrector used?
   # NEW ISSUE: clarify the mathematical details behind the
   ZEMLIN_TABLEAU(NXprocess):
```

Comment:
Why does this have such a concrete name when it is anyway uppercase and thus to be overwritten? Just for namefitting?

Reply:
has been changed to zemlin_tableauID in #251